b3sum/src/main.rs

```diff
@@ -25,6 +25,7 @@ struct Inner {
     /// Files to hash, or checkfiles to check
     ///
     /// When no file is given, or when - is given, read standard input.
+    /// The - argument represents standard input.
```

Comment:
Thanks for the PR. I like the other changes, but this one feels a little redundant with the line above. What do you think?